dblookupext/historyRepository_test.go

```diff
@@ -94,12 +94,12 @@ func TestNewHistoryRepository(t *testing.T) {
 	require.NotNil(t, repo)
 }
 
-func TestHistoryRepository_RecordBlock_InvalidBlockRoundByNonceStorer_ExpectError(t *testing.T) {
+func TestHistoryRepository_RecordBlock_InvalidBlockRoundByHashStorer_ExpectError(t *testing.T) {
```

Comment:
too many _ in name. To align with the rest of the project, please rename this to: 
```
TestHistoryRepository_RecordBlockInvalidBlockRoundByHashStorerExpectError
```

Reply:
Renamed it to be consistent with the rest of the tests